### ISSUE TITLE
Renaming LiquidPledgingBase: idxDelegate vs idDelegate & projectID

### ISSUE THREAD

**Issue by GriffGreen** [Giveth/liquidpledging#47]
1. projectId --> idProject
projectId is listed just a couple times where as idProject is listed a bunch, please change!

2. idxDelegate vs idDelegate

This is sooooo confusing, and I am fairly certain it is used inconsistently throughout LiquidPledging base

If we follow the pattern laid out with idProject and idGiver, idDelegate should refer to the id number in the admins array... but we call that idxDelegate in `addDelegate()`, `updateDelegate()`, `getDelegateIdx()` (which has a messed up name cause it has the idxDelegate as a parameter!) 

Some times idxDelegate is used the way i would expect it to be used, as the index number for its authority in the delegation chain... this happens in `getPledgeDelegate()` and in the function name`getDelegateIdx()`;-)

I would love to hear comments on this, but i see 2 proposals:

My preferred (and easier):
Check all the smart contracts and tests for consistent use of:
idDelegate: the id number in the admin array
idxDelegate: the index number in delegateChain array, different for each specific Pledge 

Option 2:

Check all the smart contracts and tests for consistent use of:
idDelegate: the index number in delegateChain array, different for each specific Pledge
idxDelegate: the id number in the admin array

And change:
idGiver -> idxGiver
idProject -> idxProject

**Comment by Quazia** [comment]
Would it really be that big of a deal to just do delegateIndex or something like that?

**Comment by perissology** [comment]
Option 1 is how it should be. We just need to reconcile any inconsistencies.

**Comment by perissology** [comment]
> Would it really be that big of a deal to just do delegateIndex or something like that?

No, but that's not any more informative as both refer to indexs. 1 in the `admins` array and the other in the `delegationChain` array for an individual pledge

**Comment by perissology** [comment]
I think this is done, just need to verify the following

> Check all the smart contracts and tests for consistent use of:
idDelegate: the id number in the admin array
idxDelegate: the index number in delegateChain array, different for each specific Pledge
